fed/mosaic-cell/src/main/java/org/eclipse/mosaic/fed/cell/data/SimulationData.java

```diff
@@ -184,7 +184,9 @@ public double getSpeedOfNode(String nodeId) {
         if (node != null && node.speed != null) {
             return node.speed;
         } else {
-            log.warn("Tried getting speed of node {}, but isn't there.", nodeId);
+            if (log.isDebugEnabled()) {
+                log.debug("Tried getting speed of node {}, but isn't there.", nodeId);
+            }
```

Comment:
`log.debug("Tried getting speed of node {}, but isn't there.", nodeId);` is sufficient. We only would need the if condition, if the formatting of the log message would be hard to compute or have many strings to concatenate.. but here we just pass raw strings to the method call.

Reply:
Removed if-clause.